core/cache/watcher.go

```diff
@@ -146,6 +148,188 @@ func (w *ConfigWatcher) configChanged(topic string, value interface{}) {
 	w.notify()
 }
 
+type MachineAppLXDProfileWatcher struct {
```

Comment:
This is a very particular watcher, probably worthy of its own file.

Reply:
moving